stalkbook/views.py

```diff
@@ -1,8 +1,24 @@
+import flickrapi
```

Comment:
this is  a third party library. And should go in different import group. This should go after `from django.conf import settings`